spec/unit/provider/package/rpm_spec.rb

```diff
@@ -56,6 +56,14 @@
     context "installation exists" do
       let(:stdout) { "ImageMagick-c++ 6.5.4.7-7.el6_5" }
 
+      it "should accept uri formed source" do
```

Comment:
I think you should test http, https, ftp, file here.

You should definitely test the casing